bitrise.yml

```diff
@@ -0,0 +1,326 @@
+format_version: 5
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  # Define it in .bitrise.secrets.yml
```

Comment:
or in secrets on Bitrise. I think we have a a unified message for this. Please use that.